quill-sql/src/main/scala/io/getquill/context/sql/idiom/SqlIdiom.scala

```diff
@@ -257,13 +259,17 @@ trait SqlIdiom {
 
   implicit def propertyShow(implicit valueShow: Show[Value], identShow: Show[Ident], strategy: NamingStrategy): Show[Property] =
     Show[Property] {
-      case Property(ident, "isEmpty")      => s"${ident.show} IS NULL"
-      case Property(ident, "nonEmpty")     => s"${ident.show} IS NOT NULL"
-      case Property(ident, "isDefined")    => s"${ident.show} IS NOT NULL"
       case Property(Property(ident, a), b) => s"${ident.show}.$a$b"
       case Property(ast, name)             => s"${scopedShow(ast)}.${strategy.column(name)}"
     }
 
+  implicit def optionPropertyShow(implicit strategy: NamingStrategy): Show[OptionProperty] = Show[OptionProperty] {
+    case OptionProperty(ast, "get")       => ast.show
+    case OptionProperty(ast, "isEmpty")   => s"${ast.show} IS NULL"
+    case OptionProperty(ast, "nonEmpty")  => s"${ast.show} IS NOT NULL"
+    case OptionProperty(ast, "isDefined") => s"${ast.show} IS NOT NULL"
```

Comment:
What do you think about moving those strings to a `OptionOperator` class, just like `SetOperator`?